### PR TITLE
Fixed issue in setup for Windows contributors in README.md

### PR DESCRIPTION
**Pull request by ShubhamAvasthi**
### Description

Fixes issue in setup for Windows contributors in README.md

### Type of Change:

- Documentation

### How Has This Been Tested?
Checked the correctness of the change on my computer.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Setup for developers (Windows)
     $ CREATE USER <anyname you want e.g systers> WITH PASSWORD 'your password';
     $ CREATE DATABASE systersdb;
     $ \c systersdb;
-    $ GRANT ALL PRIVILEGES ON systersdb TO <username created above>;
+    $ GRANT ALL PRIVILEGES ON DATABASE systersdb TO <username created above>;
```
Comment (swatisingh45) [inline README.md]: I think, considering to the previous context, it is automatically referring to a Database, need not mention explicitly.

Comment (ShubhamAvasthi) [reply README.md]: The old command does not work on Windows. I have tried it.

**Comment by sammy1997** [comment]
@ShubhamAvasthi Are you still working on this?

**Comment by sammy1997** [comment]
Closing the PR as the developer didn't respond

**Pull request closed without merging**